CalendarApp/BookingRoom/Controllers/PostEventController.cs

```diff
@@ -17,10 +20,21 @@ public PostEventController(BookingService bookingService)
 
         }
         [HttpPost]
-        public HttpResponseMessage EventPost(CalendarEvent eventPost)
+        public HttpResponseMessage PostEvent(CalendarEvent eventPost)
```

Comment:
Is there reason to use `HttpPost`-attribute and name method `PostEvent` simultaneously?